config/feed-variants/20220920-variant-a.json

```diff
@@ -0,0 +1,151 @@
+{
+  "max_days_since_published": 15,
+  "description": "Copy of 20220907-variant-c with Higher Starting matching_positive_tags_intersection_count, more steps",
+  "order_by": "final_order_by_random_weighted_to_score",
+  "reseed_randomizer_on_each_request": false,
+  "levers": {
+    "daily_decay": {
+      "cases": [
+        [0, 1],
+        [1, 0.99],
+        [2, 0.985],
+        [3, 0.98],
+        [4, 0.975],
+        [5, 0.97],
+        [6, 0.965],
+        [7, 0.96],
+        [8, 0.955],
+        [9, 0.95],
+        [10, 0.945],
+        [11, 0.94],
+        [12, 0.935],
+        [13, 0.93],
+        [14, 0.925]
+      ],
+      "fallback": 0.9
+    },
+    "comments_count_by_those_followed": {
+      "cases": [
+        [0, 0.95],
+        [1, 0.98],
+        [2, 0.99]
+      ],
+      "fallback": 0.93
+    },
+    "comments_count": {
+      "cases": [
+        [0, 0.5],
+        [1, 0.6],
+        [2, 0.66],
+        [3, 0.7],
+        [4, 0.75],
+        [5, 0.8],
+        [6, 0.85],
+        [7, 0.88],
+        [8, 0.9],
+        [9, 0.92],
+        [12, 1.0],
+        [18, 1.0],
+        [22, 1.0],
+        [25, 1.0],
+        [30, 1.0],
+        [35, 1.0],
+        [40, 1.0],
+        [45, 1.0]
+      ],
+      "fallback": 0.98
+    },
+    "featured_article": { "cases": [[1, 1]], "fallback": 0.85 },
+    "latest_comment": {
+      "cases": [
+        [0, 1],
+        [1, 0.9988]
+      ],
+      "fallback": 0.988
+    },
+    "matching_negative_tags_intersection_count": {
+      "cases": [
+        [0, 1],
+        [1, 0.5],
+        [2, 0.4],
+        [3, 0.3],
+        [4, 0.2]
+      ],
+      "fallback": 0
+    },
+    "matching_positive_tags_intersection_points": {
+      "cases": [
+        [0, 0.2],
+        [1, 0.35],
+        [2, 0.5],
+        [3, 0.65],
+        [4, 0.8],
+        [5, 0.88],
+        [6, 0.9],
+        [7, 0.92],
+        [8, 0.95],
+        [9, 0.98]
```

Comment:
This whole section is off? should it have been
```
       [0, 0.45],
       [1, 0.5],
       [2, 0.55],
       [3, 0.6],
       [4, 0.65],
       [5, 0.7],
       [6, 0.75],
       [7, 0.8],
       [8, 0.9],
       [9, 0.95]
```

Reply:
Oh good call @maestromac — I actually just didn't _save_ the file I was editing 🙃